Description

```diff
@@ -0,0 +1,47 @@
+# ETH•TORCH :unicorn: :fire:
+
+### Objective
+
+Create a symbolic ritual where the last Devcon city/country host brings its culture and passes the Eth values to the new city/country host of Devcon creating links among faraway communities.
+
+### Inspiration
+
+We are inspired on the original Olympics values: 
+- Encourage Effort (Excellence)
+- Preserve Human Dignity (Respect)
+- Develop Harmony (Friendship)
+unifying them with the Devcon objective of caring about technology impact & the Infinite Garden & Ethos of Ethereum as an equal opportunity technology for coordination and prosperity.
+
+### Activity
+
+This activity will be held ideally in 2 times:
+
+#### 1. Passing the torch
+
+We would like to create a special moment in the opening ceremony or the closing ceremony where one or two representatives of the last Devcon pass the torch vibes to the new one.
```

Comment:
```suggestion
We would like to create a special moment where one or two representatives of the last Devcon pass the torch vibes to the new one.
```

Reply:
opening and closing ceremony are not yet planned and skylar wants to under-promise and maybe then over-deliver - so we want to rather remove this specific here